lms/djangoapps/branding/api.py

```diff
@@ -170,22 +171,27 @@ def _footer_social_links():
     return links
 
 
-def _footer_connect_links():
+def _footer_connect_links(language="en"):
     """Return the connect links to display in the footer. """
+    links = OrderedDict([
+        ("blog", (marketing_link("BLOG"), _("Blog"))),
+        ("contact", (_build_support_form_url(), _("Contact Us"))),
+        ("help-center", (settings.SUPPORT_SITE_LINK, _("Help Center"))),
+        ("media_kit", (marketing_link("MEDIA_KIT"), _("Media Kit"))),
+        ("donate", (marketing_link("DONATE"), _("Donate"))),
+        ])
+
+    if language.startswith("es"):
+        del links["media_kit"]
```

Comment:
I believe it would be better if you just add these links `if language == settings.LANGUAGE_CODE`, no need for deletion.